app/controllers/qa_to_content_controller.rb

```diff
@@ -0,0 +1,70 @@
+require 'gds_api/helpers'
+
+# This class currently supports only one question
+# If there is a need for multiple questions, this class needs to be modified
+
+class QaToContentController < ApplicationController
+  layout "finder_layout"
+  include GdsApi::Helpers
+
+  def show
+    return error_not_found unless ENV["FINDER_FRONTEND_ENABLE_QA_TO_CONTENT"]
+
+    if params[first_question["id"]].present?
+      redirect_to_guidance params[first_question["id"]]
+    else
+      render 'qa_to_content/show'
+    end
+  end
+
+private
+
+  def qa_config
+    @qa_config ||= YAML.load_file("lib/#{request.path.tr('-', '_')}.yaml")
```

Comment:
I wonder if we need some protection here in case someone puts `../../` in the URL and manages to read a YAML file from somewhere else. I imagine there isn't much of a risk here given it has to be a valid YAML file, but I believe YAML files can contain executable code blocks.

Reply:
I think it will be fine because the routes are defined and if the user adds any routes which don't match it will through a 404.

Unless we start allowing dynamic routes - in which case, what you are saying might be possible.

Reply:
Good point!